client.go

```diff
@@ -987,6 +987,20 @@ func (client *gocloak) GetRealm(token string, realm string) (*RealmRepresentatio
 	return &result, nil
 }
 
+// GetRealms returns top-level representation of all realms
+func (client *gocloak) GetRealms(token string) ([]RealmRepresentation, error) {
```

Comment:
Please return the list of pointers, not objects: `[]*RealmRepresentation`

You can take a look at any function what returns a list, for example: https://github.com/Nerzal/gocloak/blob/95a49f04b887503dbfc101150c02e843ee1b6851/client.go#L724